lib/js/protocol/enums/FrameType.js

```diff
@@ -37,43 +37,35 @@ import { Enum } from '../../util/Enum.js';
  * @property {Object} MAP
```

Comment:
Please prefix all references of `MAP` with an underscore to indicate that it is a private property.